### PR TITLE
Publish webhook blog post

### PR DESCRIPTION
**Pull request by smoreface**
End-to-end webhook blog post, verified and double checked!

### CODE REVIEW

File: docs/source/blog/webhook-blog.md [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+```bash
+echo $ROLE_ARN
+```
+You need this RoleArn for the SageMaker pipeline definition. Copy the output and save it somewhere secure. 
+
+Now you're ready to define the SageMaker pipeline. 
+
+### Create a SageMaker pipeline
+
+In Amazon SageMaker, create a model training pipeline with a pretrained ResNet50 semantic image segmentation model. using the source images and annotations created in Label Studio and stored in Amazon S3. The pipeline runs the preprocessing script to split the raw and annotation data from the S3 bucket into training and validation sets, retrains the ResNet50 model, and outputs the results to the S3 bucket prefixed with `model`.
```
Comment (smoreface) [inline docs/source/blog/webhook-blog.md:238]: @KonstantinKorotaev please check this description. is anything missing that would be useful to know?

File: docs/source/blog/webhook-blog.md [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+```
+You need this RoleArn for the SageMaker pipeline definition. Copy the output and save it somewhere secure. 
+
+Now you're ready to define the SageMaker pipeline. 
+
+### Create a SageMaker pipeline
+
+In Amazon SageMaker, create a model training pipeline with a pretrained ResNet50 semantic image segmentation model. using the source images and annotations created in Label Studio and stored in Amazon S3. The pipeline runs the preprocessing script to split the raw and annotation data from the S3 bucket into training and validation sets, retrains the ResNet50 model, and outputs the results to the S3 bucket prefixed with `model`. 
+
+This example Amazon SageMaker pipeline definition has been updated to match the S3 bucket names that you created earlier, but you need to update it with the `RoleArn` of the role that you set up in previous steps. Replace `$SageMakerRoleArn` with the actual `RoleArn` value that you saved.
```
Comment (smoreface) [inline docs/source/blog/webhook-blog.md:240]: Does `\"RoleArn\": \"ROLE_FOR_SAGEMAKER_PIPELINE\"` also need to be replaced with the actual RoleArn value? Yes, correct? 
@KonstantinKorotaev

File: docs/source/blog/webhook-blog.md [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+
+<br/><img src="/images/webhook-blog/sagemaker-webhooks.png" alt="The same diagram as earlier showing data flow from bird images to an S3 bucket, flowing to Label Studio, which outputs annotations to the S3 bucket and sends a webhook to the Amazon API Gateway, which passes that information to the AWS Lambda function, which triggers the Amazon SageMaker training pipeline when the annotation count reaches at least 16, then outputs the resulting model to the S3 bucket." class="gif-border" width="800px" height="661px" />
+
+After you create at least 16 annotations, the Lambda function invokes the SageMaker pipeline that you created and starts processing the data using the preprocessing script and training the ResNet50 model in the SageMaker pipeline.
+
+To see the status of the pipeline, you need the pipeline execution ARN generated when the Lambda function executes the pipeline. After you retrieve the execution ARN, you can run the following from the command line:
```
Comment (smoreface) [inline docs/source/blog/webhook-blog.md:522]: @KonstantinKorotaev / @farioas any idea how/where they could get the execution arn?

**Comment by smoreface** [comment]
Hoping to merge and publish this first thing tomorrow, please take a final look everyone! 
@niklub / @KonstantinKorotaev / @farioas

File: docs/source/blog/Automate-your-ML-pipeline-with-Label-Studio-and-Amazon-SageMaker.md [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+                original_height = anno['result'][0]['original_height']
+                filepath = os.path.join(input_data_path,
+                                        os.path.basename(urlparse(anno['task']['data']['image']).path))
+                pil_use(poly, original_width, original_height, filepath.split('.')[0] + ".png")
+            except:
+                print(f"{p} has invalid annotations.")
+        os.remove(p)
+    print('Annotation files: ' + str(os.listdir(input_data_path)))
+    for image in os.listdir(input_data_path):
+        target_set = 'train_1' if random.randint(0, 99) < split_ratio else 'validation'
```
Comment (KonstantinKorotaev) [inline docs/source/blog/Automate-your-ML-pipeline-with-Label-Studio-and-Amazon-SageMaker.md:138]: Please change to random.randint(0, 99) **>** split_ratio

**Review by KonstantinKorotaev — APPROVED** [review]

**Review by farioas — APPROVED** [review]

**Review by niklub — APPROVED** [review]
